autoscheduler/scraper/models/course.py

```diff
@@ -1,4 +1,18 @@
 from django.db import models
 
-class Course(models.Model):
-    """ Describe model here """
+class Course(models.Model): # Directly from GBS:w
+    id = models.CharField(max_length=10, primary_key=True) # Could be dept+course_num
+    dept = models.CharField(max_length=4, db_index=True) # CSCE
+    course_num = models.CharField(max_length=5, db_index=True) # i.e. 221
+    title = models.CharField(max_length=100) # Course title, i.e. "Data Structres & Algorithms"
+    description = models.TextField(null=True, blank=True)
+    prerequisites = models.TextField(null=True, blank=True)
+    corequisites = models.TextField(null=True, blank=True)
+    cd = models.CharField(max_length=3)#yes or no. states whether course is a cd credit
+    icd = models.CharField(max_length=3)#yes or no. states whether course is an icd credit
+    core_curriculum = models.CharField(max_length=32)#tag specifying what it counts as ex) 'Creative Arts' or 'American History'. If not a core curriculum class it can be blank or 'NA' or something
+    credit_hours = models.FloatField(null = True)#Number of credit hours the course is worth. I didn't really see a point in the min/max credits because I've yet to see the same course with variable credit hours in the same semester (probably because I'm a freshman though).

```

Comment:
> I didn't really see a point in the min/max credits because I've yet to see the same course with variable credit hours in the same semester (probably because I'm a freshman though).

Just by the way, comments like these are much more appropriate to include in the pull request/code review as a comment on this line(like I'm doing now), rather than being an actual comment in the code - as they won't really pertain to anyone else in a few weeks.

Also idk if this helps, but I literally did this *exact* same thing at my first internship this summer for my first pull request/code review, so I 100% know where you're coming from 😃 . What I typically do if I want to remember a comment is have them in the file like you have it now(except on a separate line), then when I'm committing them or about to make a pull request, copy them to a text file then add them as a comment on the code review.

Reply:
Ok. Thanks for the tip. Will do next time I make a pull request.